dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -992,36 +992,55 @@ that the Airflow works as you expected.
 
 Once the vote has been passed, you will need to send a result vote to dev@airflow.apache.org:
 
-Subject:
+In both subject and message update DATE OF RELEASE, FIRST/LAST NAMES and numbers). In case
+some providers were  excluded, explain why they were excluded and what is the plan for them
+(otherwise remove the optional part of the message). There are two options for releasing
+the next RC candidates:
+
+* They will be released as an ad-hoc release with accelerated vote
+  period on their own (when there are not many changes to other providers in the meantime and when
+  we have a small bugfix for the providers that we want to release quickly.
+
+* They will be included together with the next wave of releases (our tooling
+  supports automated calculation of RC version for candidates for the next wave of releases that
+  already had earlier RCs.
+
+Email subject:
 
 ```
 [RESULT][VOTE] Airflow Providers - release of DATE OF RELEASE
 ```
 
-Message:
+Email content:
 
 ```
 Hello,
 
-Apache Airflow Providers (based on RC1) have been accepted.
+Apache Airflow Providers prepared on DATE OF RELEASE have been accepted.
 
 3 "+1" binding votes received:
-- Jarek Potiuk  (binding)
-- Kaxil Naik (binding)
-- Tomasz Urbaszek (binding)
+- FIRST LAST  (binding)
+- FIRST LAST  (binding)
+- FIRST LAST  (binding)
 
+[optional] The providers PROVIDER, PROVIDER have been excluded from the release.
+This is due to REASON HERE.
+The next RC candidates for those providers will be released [in the next wave
+of providers] or [as an ad-hoc release on their own with accelerated vote period].
 
-Vote thread:
-https://lists.apache.org/thread.html/736404ca3d2b2143b296d0910630b9bd0f8b56a0c54e3a05f4c8b5fe@%3Cdev.airflow.apache.org%3E
+2 "+1" non-binding votes received:
+
+- FIRST LAST
+- FIRST LAST
```

Comment:
Maybe you should move this to below the binding votes.

Reply:
Yes.